<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>462</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
background-color : black;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="inputContainer" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#inputContainer{
/*
border : 1px solid orange;
*/
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_inputs">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#label_inputs
{
background-color : rgb(247, 250, 252);
border-radius : 2px;
border : 1px solid lightgrey;
}</string>
        </property>
        <property name="text">
         <string>Inputs</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="outerouter" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#outerouter{
/*
border : 1px solid black ;
*/
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="outer" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget#outer{
border-radius : 5px;
background-color : LightSlateGray;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="radioContainer" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#radioContainer{

background-color : lightgrey;
border-radius : 2px;
}</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="radioButton_generate">
                 <property name="text">
                  <string>Generate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_fromFile">
                 <property name="text">
                  <string>From File</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QGridLayout" name="gridLayout_7">
                  <property name="leftMargin">
                   <number>9</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_searchElement">
                    <property name="text">
                     <string>Search for element</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="2">
                   <widget class="QSpinBox" name="spinBox_searchElement">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="minimum">
                     <number>-10000</number>
                    </property>
                    <property name="maximum">
                     <number>10000</number>
                    </property>
                    <property name="value">
                     <number>5</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="inputSelectionContainer" native="true">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#inputSelectionContainer{
background-color : lightgrey;
border-radius : 2px;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="generateContainer" native="true">
                 <property name="styleSheet">
                  <string notr="true">QWidget#generateContainer{
/*
border : 1px solid black ;
*/
}</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="5" column="1">
                   <widget class="QLabel" name="label_upperLimit">
                    <property name="text">
                     <string>Upper Limit Num</string>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="2">
                   <widget class="QPushButton" name="pushButton_generate">
                    <property name="text">
                     <string>Generate</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="2">
                   <widget class="QSpinBox" name="spinBox_upperLimit">
                    <property name="minimum">
                     <number>-10000</number>
                    </property>
                    <property name="maximum">
                     <number>10000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLabel" name="label_lowerLimit">
                    <property name="text">
                     <string>Lower Limit Num</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="2">
                   <widget class="QSpinBox" name="spinBox_numElements">
                    <property name="maximum">
                     <number>10000</number>
                    </property>
                    <property name="value">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QRadioButton" name="radioButton_random">
                    <property name="text">
                     <string>Random</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLabel" name="label_numElements">
                    <property name="text">
                     <string>Num of Elements</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="2">
                   <widget class="QSpinBox" name="spinBox_lowerLimit">
                    <property name="minimum">
                     <number>-10000</number>
                    </property>
                    <property name="maximum">
                     <number>10000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QRadioButton" name="radioButton_sequential">
                    <property name="text">
                     <string>Sequential</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="2">
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="2" column="1">
                   <widget class="QWidget" name="widget_3" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>2</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border : 1px solid black;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="fromFileContainer" native="true">
                 <property name="styleSheet">
                  <string notr="true">QWidget#fromFileContainer{
/*
border : 2px solid black ;
*/
}</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="2" column="0">
                   <widget class="QLineEdit" name="label_fileoutput"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="pushButton_openFile">
                    <property name="text">
                     <string>Open File</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="output" native="true">
           <property name="styleSheet">
            <string notr="true">QWidget#output{

border-radius : 5px;
background-color : LightSlateGray;
}</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QTextBrowser" name="textBrowser"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_preview">
              <property name="styleSheet">
               <string notr="true">QWidget#label_preview
{
background-color : rgb(247, 250, 252);
border-radius : 2px;
border : 1px solid lightgrey;
}</string>
              </property>
              <property name="text">
               <string>Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="buttonContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#buttonContainer{

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="widget_2">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-right :0;</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>widget_2</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_2</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
